consensus/polybft/polybft.go

```diff
@@ -215,28 +195,33 @@ func (p *Polybft) Initialize() error {
 	p.state = stt
 	p.validatorsCache = newValidatorsSnapshotCache(p.config.Logger, stt, p.consensusConfig.EpochSize, p.blockchain)
 
-	return nil
-}
+	// set pbft topic, it will be check if/when the bridge is enabled
```

Comment:
```suggestion
	// set consensus engine topic, it will be check if the bridge is enabled
```

Reply:
Comment was not good at all https://github.com/0xPolygon/polygon-edge/pull/865/commits/e9f82c8aa934344e1b727726a10daa473becae37